File: 0204-count-primes/0204-count-primes.cpp
class Solution {
public:
    int countPrimes(int n) {
        bool p[n+1];
        memset(p,true,sizeof(p));
        for(long long i=2;i<n;++i){
            if(p[i]){
                for(long long j = i*i;j<n;j+=i){
                    p[j]=false;
                }
            }
        }
        int ans =0;
        for(long long i=2;i<n;++i){
            if(p[i])++ans;
        }
        return ans;
    }
};

File: 0204-count-primes/0204-count-primes.py
class Solution:
    def countPrimes(self, n: int) -> int:
        prime_arr = [True for x in range (n+1)]
        for i in range (2,n):
            if prime_arr[i]==True:
                for j in range ( i*i,n,i):
                    prime_arr[j] = False
        ans = 0
        # print(prime_arr)
        for i in range(2,n):
            if prime_arr[i]==True:
                ans+=1
        return ans